### PR TITLE
feat(client): disable admin ui and graphql on resource settings page

### PR DESCRIPTION
**Pull request by overbit**
Close: #8619

## PR Details

Disable adminui and graphql from resource settings in client  

## PR Checklist

- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGenerationSettings.tsx [hunk 1]
```diff
@@ -19,7 +19,17 @@ const CreateGenerationSettings: React.FC<WizardStepProps> = ({
   trackWizardPageEvent,
 }) => {
   useEffect(() => {
-    if (!formik.values.generateGraphQL) {
+    if (
+      formik.values.codeGenerator !== "NestJS" &&
```
Comment (morhag90) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGenerationSettings.tsx:23]: @overbit please use the models.EnumCodeGenerator instead

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGenerationSettings.tsx [hunk 2]
```diff
@@ -44,6 +54,7 @@ const CreateGenerationSettings: React.FC<WizardStepProps> = ({
           <ImageLabelToggle
             name="generateGraphQL"
             image={graphql}
+            disabled={formik.values.codeGenerator !== "NestJS"}
```
Comment (morhag90) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGenerationSettings.tsx:57]: @overbit  here too : models.EnumCodeGenerator

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGenerationSettings.tsx [hunk 3]
```diff
@@ -58,7 +69,10 @@ const CreateGenerationSettings: React.FC<WizardStepProps> = ({
           <ImageLabelToggle
             name="generateAdminUI"
             image={adminUI}
-            disabled={!formik.values.generateGraphQL}
+            disabled={
+              !formik.values.generateGraphQL &&
```
Comment (morhag90) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGenerationSettings.tsx:73]: @overbit  || ??

**Comment by mulygottlieb** [comment]
@yuval-hazaz  please review as well

**Review by yuval-hazaz — APPROVED** [review]

**Comment by yuval-hazaz** [comment]
I added a message on both pages and changed the order of the fields 

image

**Pull request merged**